doc/modules/cassandra/pages/developing/cql/SASI.adoc

```diff
@@ -519,7 +519,7 @@ execution.
 
 During the analysis phase,
 https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/index/sasi/plan/QueryPlan.java[`QueryPlan`]
-converts from Cassandra's internal representation of `IndexExpression`s,
+converts from Cassandra's internal representation of ``IndexExpression``s,
```

Comment:
Same for this one. Please check all around.

Reply:
![SASI_3](https://github.com/user-attachments/assets/1bfe4119-8dad-4d5d-a017-79c8eae74bee)

fix renders properly after antora build